lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
       <nav class="page-content-nav">
         <a href="" class="nav-item is-active">My Teams</a>
-        <a href="" class="nav-item">Browse All Teams</a>
+        <a href="" class="nav-item">Browse</a>
       </nav>
 
       <div class="page-content-main">
-        <p>Blah blah blah. Go Team!</p>
+
+        <div class="search-tools">
+          <span class="search-count">Viewing 12 of 24 topics</span>
+          <span class="search-sort">sorted by
+          <select name="search-sort">
+            <option>number of teams</option>
+            <option>alphabetically</option>
+            <option>latest activity</option>
+          </select>
+        </div>
+
+
+        <div class="card square-card topic-card has-pennant">
+          <div class="card-core-wrapper">
```

Comment:
For structural/cosmetic UI wrappers, we tend to prefix class names with "wrapper-" for better scanning and understanding of the element's purpose. Mind changing ``card-core-wrapper`` to ``wrapper-card-core``?